barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_group.test.cpp

```diff
@@ -1843,6 +1846,97 @@ TYPED_TEST(CycleGroupTest, MixedLengthScalarsIsNotSupported)
     EXPECT_THROW_OR_ABORT(cycle_group_ct::batch_mul(points, scalars), "Assertion failed: (s.num_bits() == num_bits)");
 }
 
+/**
+ * @brief Test batch_mul with cycle_scalar created from create_from_bn254_scalar
+ * @details This tests the create_from_bn254_scalar constructor path which performs in-field validation
+ * against bn254::fr::modulus during construction via split_unique.
+ */
+TYPED_TEST(CycleGroupTest, TestBatchMulWithBn254Scalar)
```

Comment:
Just ensuring here that we have basic tests for batch_mul where the scalars are coming from BN254 (`create_from_bn254_scalar()`) and u256 (`from_u256_witness()`)